automatic/vscode-insiders.install/tools/ChocolateyInstall.ps1

```diff
@@ -0,0 +1,33 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+. $toolsPath\helpers.ps1
+
+$softwareName = 'Microsoft Visual Studio Code Insiders'
+$version = '1.46.0'
+if ($version -eq (Get-UninstallRegistryKey "$softwareName").DisplayVersion) {
+  Write-Host "VS Code Insiders $version is already installed."
+  return
+}
+
+$pp = Get-PackageParameters
+Close-VSCodeInsiders
```

Comment:
Please move the closing of vscode into a `chocolateyBeforeModify.ps1` file (this file runs before upgrade and before uninstall).

*Yes, I am aware we should do that for the original vscode package as well*